jina/peapods/runtimes/gateway/grpc/__init__.py

```diff
@@ -42,16 +41,16 @@ async def async_setup(self):
             ]
         )
         self.zmqlet = AsyncZmqlet(self.args, logger=self.logger)
-        jina_pb2_grpc.add_JinaRPCServicer_to_server(
-            GRPCPrefetchCall(self.args, self.zmqlet), self.server
-        )
+        self._prefetcher = GRPCPrefetchCall(self.args, self.zmqlet)
+        jina_pb2_grpc.add_JinaRPCServicer_to_server(self._prefetcher, self.server)
         bind_addr = f'{self.args.host}:{self.args.port_expose}'
         self.server.add_insecure_port(bind_addr)
         await self.server.start()
 
     async def async_cancel(self):
         """The async method to stop server."""
         await self.server.stop(0)
+        await self._prefetcher.servicer.close()
```

Comment:
```suggestion
        await self._prefetcher.close()
```
There's no `servicer` object in `_prefetcher` right?